System/Random.hs

```diff
@@ -1057,32 +1059,32 @@ class Random a where
 
   --{-# INLINE randomM #-}
   randomM :: MonadRandom g s m => g s -> m a
-  -- default randomM :: (MonadRandom g m, Uniform a) => g -> m a
-  -- randomM = uniform
+  default randomM :: (MonadRandom g s m, Uniform a) => g s -> m a
+  randomM = uniform
```

Comment:
Default implementation should be removed. It will break all custom instances of `Random` otherwise. In fact I think `randomM` should be removed from this class. I'll submit a separate PR for this one.

Reply:
I will revert that commit. For my benefit: how would this default implementation break custom implementations? In my mental model, the default implementation becomes irrelevant as soon as there is a custom implementation.

> In fact I think `randomM` should be removed from this class. I'll submit a separate PR for this one.

:+1:

Reply:
This is now implemented in #90